src/main/dissolve.h

```diff
@@ -31,6 +32,7 @@ class Dissolve
     private:
     // Reference to CoreData
     CoreData &coreData_;
+    SerializablePairPotential pairPotential_;
```

Comment:
This is a bit misleading as named.
```suggestion
    SerializablePairPotential serializablePairPotentials_;
```